introduction.md

```diff
@@ -0,0 +1 @@
+I'm from Mexico, I work as a frontend developer. ✌🏼
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.